WEB_SERVICE/DNS/_HIGH/DNS_INFORMATION_DISCLOSURE/meta.json

```diff
@@ -0,0 +1,62 @@
+{
+  "risk_rating": "high",
```

Comment:
- Medium Risk: If the disclosure only includes basic public-facing information (e.g., web and mail server records).
- High Risk: If it reveals internal infrastructure, exposes AXFR, or supports further targeted attacks.

Reply:
If you read the description you will see that its about the `API Keys and Passwords` and private ips